README.md

```diff
@@ -382,6 +382,55 @@ Authentication schemes defined for the API:
   - TIMECARDS_WRITE: POST, PUT, and DELETE endpoints related to employee timecards
 
 
+## Pagination of V1 Endpoints
+
+V1 Endpoints return pagination information via HTTP headers. In order to obtain
+response headers and extract the `batch_token` parameter you will need to follow
+the following steps:
+
+1. Use the full information endpoint methods of each API to get the response HTTP
+Headers. They are named as their simple counterpart with a `WithHttpInfo` suffix.
+Hence `listEmployeeRoles` would be called `listEmployeeRolesWithHttpInfo`. This
+method returns a `CompleteResponse` object with the response data deserialized along
+with a helper to retrieve the token if present.
+
+2. Use `String batchToken = completeResponse.getBatchToken();`
+to extract the token and proceed to get the following page if a token is present.
+
+### Example
+
+```java
+// Import classes:
+//import .ApiClient;
+//import .ApiException;
+//import .Configuration;
+//import .CompleteResponse;
+//import .auth.*;
+//import .api.V1EmployeesApi;
```

Comment:
All examples are with commented packages. I guess it's because it's not a final example.